src/pages/api/teams/[teamId]/digests/[digestId]/index.ts

```diff
@@ -27,6 +28,34 @@ router
           error: 'Title cannot be empty',
         });
       }
+      const isFirstPublication = !digest?.publishedAt && !!req.body.publishedAt;
+
+      if (isFirstPublication) {
+        const lastDigests = await client.digest.findMany({
+          select: { title: true },
+          where: { teamId: digest?.teamId },
+        });
+
+        const lastDigestTitles = [
```

Comment:
maybe we can skip the completion if we dont have any titles?

Reply:
I guess we can even wait until team has at least 2 digests published otherwise chat gpt seems to predict it will be same title as the first one published...